ads/how-to-guides/dynamic-backdrops.md

```diff
@@ -0,0 +1,40 @@
+---
+sidebar_position: 8
+---
+
+# Dynamic backdrops
+
+Some layouts, such as L-shape and Double Box, make use of a backdrop. These can either be specified on both the monetised stream and the ad break in 2 ways:
+- `backdropURI`: Optional URI containing the default back-drop to be used during the Double Box or L-shape ads.
+- `backdropURIGamProperties`: Optional property with configuration values for a dynamic back-drop loaded via GAM to be used during the Double Box or L-shape ads. This property has priority over `backdropURI`.
+
+## Using the OptiView Ads service REST API
+
+The way to specify backdrops is through the OptiView Ads service API when creating or updating both monetised streams and adbreaks.
+
+The body of the post request for both monetised streams and ad breaks can have the following properties:
+
+```ts
+{
+  ...
+  backdropURI: string | undefined;
+  backdropURIGamProperties: {
+    iu: string;
+    sz: {
+      default: string;
+      phone: string | undefined;
+      tv: string | undefined;
+    }
+  } | undefined;
+  ...
+}
+```
+
+- `backdropURI` can optionally be used to specify a URI that returns the actual backdrop image to be used.
+- `backdropURIGamProperties` can optionally be used to specify the values required for requesting a pre-configured custom creative from GAM (See below). It follows the [specifications](https://support.google.com/admanager/answer/2623168#zippy=%2Crequired-parameters) for a tagless request to load a creative. If `backdropURIGamProperties` is not defined, you need to specify `backdropURI`.
```

Comment:
This property doesn't appear in [our REST API docs](https://optiview.dolby.com/docs/ads/api/signaling/create-monetized-stream/). Can we update our REST API too?